src/screens/SearchScreen.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import { ScrollView, StyleSheet } from 'react-native';
+
+import EditScreenInfo from 'components/EditScreenInfo';
+import { Text, View } from 'components/Themed';
+import { useEffect, useState } from 'react';
+import { getAllTrees } from 'src/database/firebase';
+import { Tree } from '@types';
+import SearchCard from 'src/components/SearchCard';
+import { Searchbar } from 'react-native-paper';
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    alignItems: 'center',
+    justifyContent: 'center',
+  },
+  title: {
+    fontSize: 20,
+    fontWeight: 'bold',
+  },
+});
+
+export default function SearchScreen() {
+  const [searchQuery, setSearchQuery] = useState('');
+  const [trees, setTrees] = React.useState<Tree[]>([]);
+  useEffect(() => {
+    async function getTrees() {
+      let data = await getAllTrees();
+      data = data.filter(
+        (tree: Tree) => (tree.name || '').toLowerCase().includes(searchQuery) ||
```

Comment:
if there's no tree.name the record shouldn't be included, right? @kennethlien

Reply:
@kennethlien if `id` is our primary key shouldn't it be an int?

Reply:
ID is our primary key, and it should be an int. This was previously to allow for existing ID's the NPO already had to not be changed (some keys were "1a"). Let's talk about whether we want to restructure today.

Reply:
Tabling this because @kennethlien will figure out restructuring later!